app/views/embassies/index.html.erb

```diff
@@ -9,33 +8,50 @@
     } %>
   </div>
 </header>
-
-<div class="block-2">
-  <div class="inner-block embassy-listing">
-
-    <span class="a-z-heading"><span class="visuallyhidden">Countries ordered from </span><h1>A&ndash;Z</h1></span>
-
-    <ol class="locations">
-      <% @embassies_by_location.each do |embassy| -%>
-      <li>
-        <h2><%= embassy.name %></h2>
-
-        <% if embassy.has_consular_service_in_location? && embassy.offices.empty? -%>
-          <p><%= embassy.text %></p>
-        <% else -%>
-          <ul>
-            <% if !embassy.has_consular_service_in_location? || embassy.has_remote_service? -%>
-            <li>
-              <p><%= embassy.text %></p>
-              <%= embassy.embassy_path %>
-            </li>
-            <% else %>
-              <%= render partial: "organisation", collection: embassy.consular_services_organisations -%>
-            <% end %>
-          </ul>
-        <% end -%>
-      </li>
-      <% end -%>
-    </ol>
+<div class="govuk-main-wrapper">
+  <div class="govuk-grid-row">
+    <div class="embassy-listing">
```

Comment:
2. I can't see really what this is doing, even after reading the stylesheet above. Removing this class doesn't change anything for me. I reckon you can remove this.